daos-stack/daos
# DAOS-6923 test: Offline Reintegration (cherry_pick 1.2)

Test-tag-hw-medium: pr,hw,medium,ib2 offline_reintegration_daily

Summary:

- Moved some more common files to osa_utils.py
- Offline reintegration test changes
    - Added multiple pool infrastructure to test multiple pools fore offline reintegration
    - Test with different object class
    - More ranks excluded and reintegrated
    - Add mechanism to test exclude, reintegration with aggregation turned on.
- Added daos container check changes.
- Add mdtest thread methods.
- Update offline drain to accomodate osa_utils.py changes.
- Handle multiple containers for OSA testing.
- Test without checksum enabled.
